app/jobs/bulk_update/analytics_job.rb

```diff
@@ -3,6 +3,8 @@
 module BulkUpdate
   class AnalyticsJob < ApplicationJob
     def perform(model:, ids:)
+      return unless FeatureService.enabled?("google.send_data_to_big_query")
```

Comment:
Make sure to only continue if enabled as it will make a call down to https://github.com/DFE-Digital/dfe-analytics/blob/main/lib/dfe/analytics/send_events.rb